### DISCUSSION TITLE
如何得到可复制的mediainfo

### DISCUSSION THREAD

**Discussion by jiezi506** [hooke007/mpv_PlayKit#413] [mpv 通用]
像mpc里媒体文件的mediainfo在一个新窗口里，可以直接全选复制出来。请问mpv可以实现么？

**Comment by hooke007** [comment]
那你为什么不直接用mediainfo

**Reply by jiezi506** [reply]
也是一个解决方案，就是有时候不想把播放窗口切到后台再去点mediainfo，还有多开的时候，容易混乱。

**Reply by hooke007** [reply]
https://mpv.io/manual/master/#command-interface-[-]]]
mpv里有run和subprocess两个命令，能调用其它程序

一个简单例子，通过powershell调用explorer，引用mpv的path属性，实现打开当前文件的路径
```
<KEY>   run powershell "explorer '/select,' ( \"${path}\" -replace '/', '\\' )"
```

**Reply by jiezi506** [reply]
谢谢H大
